templates/base.html

```diff
@@ -124,13 +124,20 @@ <h1>{% block title %}{{product_name | capitalize }} {{ os_family | capitalize}}
             <p>
               You will need to explicitly install a Java runtime environment, because Oracle's Java RPMs are incorrect and fail to register as providing a java dependency.
               Thus, adding an explicit dependency requirement on Java would force installation of the OpenJDK JVM.
-            <p>
-
-            <ul>
-              <li>2.164 (2019-02) and newer: Java 8 or Java 11</li>
-              <li>2.54 (2017-04) and newer: Java 8</li>
-              <li>1.612 (2015-05) and newer: Java 7</li>
-            </ul>
+      <p>
+          <b> LTS Version</b>
+              <ul>
+                    <li>2.346 (2022-06) and newer: Java 8 or Java 11</li>
+                    <li>2.164 (2019-02) and newer: Java 8 or Java 11</li>
+                    <li>2.54 (2017-04) and newer: Java 8</li>
+                    <li>1.612 (2015-05) and newer: Java 7</li>
+              </ul>
+                <b>Weekly Version</b>
+      <p>
+              <ul>
+                    <li>2.3xx (2022-06-28) and newer: Java 11</li>
```

Comment:
Assuming 2.350 releases as expected May 31, 2022 and there are 3 intervening releases after that, we'll be at 2.353 for the release that requires Java 11 or newer.
```suggestion
                    <li>2.353 (2022-06-21) and newer: Java 11</li>
```

Reply:
Definitely makes sense Mark, I wasn't sure what it would be exactly, so that works for me!